data/projects.yml

```diff
@@ -1,13 +1,17 @@
 projects:
   - name: Friday Hacks
-    description: talks for hackers
+    description: Interesting talks for hackers from all walks of life
     url: //www.nushackers.org/fridayhacks
+    logo: static/img/friday-hacks-logo.svg
   - name: hackerschool
-    description: workshops for hackers
+    description: Workshops about web dev, machine learning and more
```

Comment:
Can we just let this be the old text? Clicking on the descriptions will tell them the specifics of the workshops.